OpenRA.Mods.Common/Commands/DevCommands.cs

```diff
@@ -42,21 +62,8 @@ public void WorldLoaded(World w, WorldRenderer wr)
 				help.RegisterHelp(name, helpText);
 			};
 
-			register("visibility", "toggles visibility checks and minimap.");
-			register("givecash", "gives the default or specified amount of money.");
-			register("givecashall", "gives the default or specified amount of money to all players and ai.");
-			register("instantbuild", "toggles instant building.");
-			register("buildanywhere", "toggles you the ability to build anywhere.");
-			register("unlimitedpower", "toggles infinite power.");
-			register("enabletech", "toggles the ability to build everything.");
-			register("instantcharge", "toggles instant support power charging.");
-			register("all", "toggles all cheats and gives you some cash for your trouble.");
-			register("crash", "crashes the game.");
-			register("levelup", "adds a specified number of levels to the selected actors.");
-			register("playerexperience", "adds a specified amount of player experience to the owner(s) of selected actors.");
-			register("poweroutage", "causes owner(s) of selected actors to have a 5 second power outage.");
-			register("kill", "kills selected actors.");
-			register("dispose", "disposes selected actors.");
+			foreach (var command in commandHandlers)
+				register(command.Key, command.Value.Item1);
```

Comment:
### First code style fix:

This can be changed to
```csharp
foreach (var command in commandHandlers)
{
	console.RegisterCommand(name, this);
	help.RegisterHelp(command.Key, command.Value.Item1);
}
```

and the now redundant `register` deleted.

### Second code style fix:

Change the `(string, Action<string, World>)` in the dictionary definition to `(string Description, Action<string, World> Handler)` then use `help.RegisterHelp(command.Key, command.Value.Description);` here and `command.Handler(arg, world)` below.

Reply:
That is indeed way nicer.